packages/client/mutations/UpsertTeamPromptResponseMutation.ts

```diff
@@ -77,7 +80,9 @@ const UpsertTeamPromptResponseMutation: StandardMutation<
         id: teamPromptResponseId ?? clientTempId(viewerId),
         userId: viewerId,
         content,
-        plaintextContent
+        plaintextContent,
+        updatedAt: now,
+        createdAt: now
```

Comment:
+1 This causes the createdAt time to flash from `X minutes ago` to `just now` and back.  I think we should only provide this if a response doesn't already exist (i.e. when `teamPromptResponseId` is falsey).  That way, we optimistically provide a `createdAt` of `now` for new responses, and let relay use the value already in the store for updated responses.

```suggestion
        createdAt: !teamPromptResponseId ? now : undefined
```

Reply:
Yes, that's the case!